epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/madgraph/iolibs/template_files/gpu/fbridge.inc

```diff
@@ -56,6 +56,31 @@ C
          END SUBROUTINE FBRIDGESEQUENCE
       END INTERFACE
 
+C
```

Comment:
I cannot leave a comment in code that was not changed, but I would propose in the line 52 above to change to 

```INTEGER*4 CHANID(*)```

I have done this change in the new_interface_wrap branch, see https://github.com/madgraph5/madgraph4gpu/blob/b317cbfe6310e3cf3af8ed8ea09029d5341c5af6/epochX/cudacpp/gg_tt.mad/SubProcesses/fbridge.inc#L52 and this seems to work correctly.

Reply:
Hi @roiser thanks for having a look.

I think these are two different issues:
- in my PR here, channelid is still a scalar, and so this must be `CHANID` and not `CHANID(*)`
- in your upcoming PR, you will indeed need to change `CHANID` to `CHANID(*)`

Specifically, if I simply change myself `CHANID` to `CHANID(*)`. then this fails the build
```
ccache /cvmfs/sft.cern.ch/lcg/releases/gcc/12.1.0-57c96/x86_64-centos9/bin/gfortran -w -fPIC -O3 -ffast-math -fbounds-check -ffixed-line-length-132 -w -cpp -c -DMG5AMC_MEEXPORTER_CUDACPP auto_dsig1.f -I../../Source/ -fopenmp -o auto_dsig1_cudacpp.o
auto_dsig1.f:573:32:

  573 |      &      HEL_RAND, COL_RAND, CHANNEL, OUT2,
      |                                1
Error: Rank mismatch in argument 'chanid' at (1) (rank-1 and scalar)
```

I would suggest to keep this in two different PRs
- here I just add the new method and keep `CHANID`
- in your PR you will then change `CHANID` to `CHANID(*)`
Does this sound ok? Please let me know if you think this can be merged @roiser thanks

Reply:
But this PR emerged out of the work for channel id arrays. If I do not remember correctly and it was done for another purpose please merge this here and I would adapt then with the later channel id PR if needed.

Reply:
No it was indeed done for that purpose. But it does not harm to include it anyway (it does simplify one function call without multichannel support). I would include it anyway now so that there is no need for further conflict fixes etc. It makes completely sense by itself and was fully tested by itself.